portfolio/src/main/java/com/google/sps/servlets/JSONServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+
+/** Handles requests sent to the /fun-fact URL. Try running a server and navigating to /fun-fact! */
+@WebServlet("/i-like")
```

Comment:
This isn't in use anywhere. How about adding a "get fact" button to the main page that fetches this list and picks a random one?

Reply:
Done!